docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/ui/index.md

```diff
@@ -46,4 +46,14 @@ In the image below, you can see an example of a data product. It not only provid
 
 ![Event specification details](images/event-specification-details.png)
 
-If you need to edit a data product at any time, simply select it from the data products listing accessible from the main menu.
\ No newline at end of file
+If you need to edit a data product at any time, simply select it from the data products listing accessible from the main menu.
+
+#### Updating existing Data Products following the release of Source Applications
+
+Data Products created prior to the release of [Source Applications](../../organize-data-sources-with-source-applications/index.md) will need to be updated to assign Source Applications in which the events will be tracked on. The selected Source Applications will add the relevant application IDs to each of the event specifications in the data product. 
+
+![Updating existing Data Products](images/edit-existing-data-product.png)
+
+Event specifications which contain previously added application IDs will need to be updated to use the identifiers inherited from the Source Applications selected at Data Product level. This process can be done manually or you can [reach out to our team](mailto:support@snowplow.io) to help you with that.
```

Comment:
nit: I would just link to the customer support here

https://github.com/snowplow/documentation/blob/145f8cd04e251c894b4f6403bb5f73d79e1758f8/src/pages/getting-started-with-support.md?plain=1#L29
instead of adding the email address on the text....

Reply:
But it's an email address

Reply:
I would add the link to that page, but email I think could be good ok too, it is up to you